xml/file_management.xml

```diff
@@ -204,6 +204,65 @@
 
  </sect1>
 
+ <sect1 xml:id="sec-sec-prot-general-home-permissions">
+  <title>Changing home directory permissions from 775 to 700</title>
+  <para>
+   By default, home directories of users are accessible (read, execute) by
+   other members of the group <literal>users</literal>, means all other user
+   accounts on the system.
```

Comment:
simply say "by all other users of the system", as 755 allows all users, not just the users in the users group